test/vanilla/src/main/java/fixtures/bodycomplex/models/CMYKColors.java

```diff
@@ -10,9 +10,9 @@
 
 package fixtures.bodycomplex.models;
 
-import java.util.Collection;
```

Comment:
Is this the result of regenerating enums with the new generator?

Reply:
Yes. The new generator automatically sorts all imports alphabetically.